src/main/java/org/terasology/scenario/internal/events/RegionResizeEvent.java

```diff
@@ -21,10 +21,13 @@
 import org.terasology.math.geom.Vector3f;
 import org.terasology.math.geom.Vector3i;
 import org.terasology.network.ServerEvent;
+import org.terasology.scenario.internal.systems.RegionTreeSystem;
 import org.terasology.scenario.internal.ui.HubToolScreen;
 
 /**
  * Event that is called to resize a region entity
+ *
+ * It is utilizes by the {@link RegionTreeSystem}
  */

```

Comment:
About like that I think the javadocs should look like. Short explanation plus link to a class that explains more (hopefully REgiontreeSystem would explain what a region entity is.